src/app/src/components/FacilityListItemsDetailedTableRowCell.jsx

```diff
@@ -20,11 +22,50 @@ export default function FacilityListItemsDetailedTableRowCell({
     errorState,
     linkURLs,
     readOnly,
+    isRemoved,
+    handleRemoveItem,
+    removeButtonDisabled,
+    removeButtonID,
 }) {
+    const statusSection = (() => {
+        if (isRemoved) {
+            return 'REMOVED';
+        }
+
+        if (!isFunction(handleRemoveItem)) {
+            return title;
+        }
+
+        return (
+            <div
+                style={{
+                    display: 'flex',
+                    justifyContent: 'space-between',
+                    alignItems: 'center',
+                    width: '100%',
+                }}
+            >
+                <span style={{ marginRight: '5px' }}>
+                    {title}
+                </span>
+                <Button
+                    variant="contained"
```

Comment:
Consider using a stock button. The full column of blue buttons draws a lot of focus. Contrast these with the approve/reject buttons, which we absolutely need the contributor to click on. The remove buttons are an optional feature that we may want to call less attention to.

<img width="1377" alt="Screen Shot 2019-06-24 at 12 33 51 PM" src="https://user-images.githubusercontent.com/17363/60046652-b77c5280-967c-11e9-83d8-9462d02ba24a.png">


```diff
diff --git a/src/app/src/components/FacilityListItemsTableRow.jsx b/src/app/src/components/FacilityListItemsTableRow.jsx
index 55936de..8d0748e 100644
--- a/src/app/src/components/FacilityListItemsTableRow.jsx
+++ b/src/app/src/components/FacilityListItemsTableRow.jsx
@@ -113,7 +113,6 @@ const FacilityListItemsTableRow = ({
                                 {newFacility ? 'NEW_FACILITY' : status}
                             </span>
                             <Button
-                                variant="contained"
                                 color="primary"
                                 onClick={handleRemoveItem}
                                 style={{ marginLeft: '5px', marginRight: '5px' }}
```

Reply:
Sounds good -- I adjusted these buttons in 2a796e6430bf7879205973107e624f569d824495